crates/oxc_ast/src/trivia.rs

```diff
@@ -59,26 +60,124 @@ impl Deref for Trivias {
 }
 
 impl Trivias {
-    pub fn new(comments: TriviasMap, irregular_whitespaces: Vec<Span>) -> Trivias {
+    pub fn new(comments: SortedComments, irregular_whitespaces: Vec<Span>) -> Trivias {
```

Comment:
I think this method can be switched to `pub(crate)` instead. That may allow dropping a bunch of `pub` from a chain of related structs. If that's ok, I'll be happy to explore that in a followup.

Reply:
👍 Go ahead

Reply:
Nevermind, this is involved in a cross-crate call (`oxc_parser -> oxc_ast`) thus it needs to stay public.